controllers/argocd/util.go

```diff
@@ -1092,8 +1092,31 @@ func (r *ReconcileArgoCD) setResourceWatches(bldr *builder.Builder, clusterResou
 		},
 	}
 
+	// Add new predicate to delete Notifications Resources. The predicate watches the Argo CD CR for changes to the `.spec.Notifications.Enabled`
```

Comment:
Please update the comments to modify Notifications as ApplicationSet. To be changed in multiple places, please take care